src/main/java/teammates/common/datatransfer/questions/FeedbackRankRecipientsQuestionDetails.java

```diff
@@ -157,13 +172,30 @@ private String getSubmissionOptionsHtmlForRankingRecipients(int totalNumRecipien
     @Override
     public String getQuestionSpecificEditFormHtml(int questionNumber) {
 
+        boolean isMinOptionsToBeRankedEnabled = minOptionsToBeRanked != Integer.MIN_VALUE;
+        boolean isMaxOptionsToBeRankedEnabled = maxOptionsToBeRanked != Integer.MIN_VALUE;
+
         return Templates.populateTemplate(
                 FormTemplates.RANK_EDIT_RECIPIENTS_FORM,
                 Slots.QUESTION_NUMBER, Integer.toString(questionNumber),
                 Slots.OPTION_RECIPIENT_DISPLAY_NAME, "recipient",
                 Slots.RANK_PARAM_IS_DUPLICATES_ALLOWED,
                         Const.ParamsNames.FEEDBACK_QUESTION_RANKISDUPLICATESALLOWED,
-                Slots.RANK_ARE_DUPLICATES_ALLOWED_CHECKED, isAreDuplicatesAllowed() ? "checked" : "");
+                Slots.RANK_ARE_DUPLICATES_ALLOWED_CHECKED, isAreDuplicatesAllowed() ? "checked" : "",
+                Slots.RANK_IS_MIN_RECIPIENTS_TO_BE_RANKED_ENABLED, isMinOptionsToBeRankedEnabled ? "checked" : "",
+                Slots.RANK_PARAM_MIN_RECIPIENTS_CHECKBOX,
+                        Const.ParamsNames.FEEDBACK_QUESTION_RANKISMINRECIPIENTSTOBERANKEDENABLED,
+                Slots.RANK_PARAM_MIN_RECIPIENTS_TO_BE_RANKED,
+                        Const.ParamsNames.FEEDBACK_QUESTION_RANKMINRECIPIENTSTOBERANKED,
+                Slots.RANK_MIN_RECIPIENTS_TO_BE_RANKED, isMinOptionsToBeRankedEnabled
+                        ? Integer.toString(minOptionsToBeRanked) : "1",
```

Comment:
May look more readable if `isMinOptionsToBeRankedEnabled` is also on the next line. Similarly for the `isMaxOptionsToBeRankedEnabled` down there.

Reply:
@VamsiSangam take a look at the comment before this

Reply:
I mentioned "if `isMinOptionsToBeRankedEnabled` is also on the next line", which is:
```java
Slots.RANK_MIN_RECIPIENTS_TO_BE_RANKED,
        isMinOptionsToBeRankedEnabled ? Integer.toString(minOptionsToBeRanked) : "1",
```
Then again it's just cosmetic, feel free to just ignore it.

Reply:
@VamsiSangam you're targeting the wrong line... you fixed the one that doesn't need fix. It's the one above this (let me just be precise: line 207, and also line 215 down there).